README.md

```diff
@@ -2,6 +2,9 @@
 
 This is a wrapper for Jaeger Tracing to be used with Payara Platform 5.194+, as an alternative implementation of Opentracing.
 
+Version 1.2 : Works on 5.2021.1 or higher (MicroProfile 4.x)
```

Comment:
Why are we jumping from 1.0 to 1.2?

Reply:
Since we did not release 1.1-SNAPSHOT that had some improvements and this PR makes it unusable when using <5.2021.1.

But I will revert the version change.